src/css/main.scss

```diff
@@ -445,6 +445,14 @@ body {
 .has-error .col-form-label {
   color: $red-beta;
 }
+div.has-error {
+  input {
+    border: 2px solid $red-beta;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
+    border-radius: 4px;
+  }
+}
```

Comment:
why did you add those styles? Don't we have them already?

Reply:
We had them, but in the putaway modal (split line button) styles should be applied on .has-error > input (this styles didn't work before). I am adding this tag to styles below, but border-radius has to be added separately.